docroot/themes/contrib/civictheme/civictheme_library/components/00-base/_variables.components.scss

```diff
@@ -964,156 +889,154 @@ $civictheme-content-dark-background-color: inherit !default;
-$civictheme-footer-light-collapsible-border-color: civictheme-color('neutral', 5) !default;
-$civictheme-footer-dark-color: civictheme-color('neutral', 1) !default;
-$civictheme-footer-dark-background-color: civictheme-color('primary', 5) !default;
-$civictheme-footer-dark-border-color: civictheme-color('primary') !default;
-$civictheme-footer-dark-outline-color: civictheme-color('outline') !default;
-$civictheme-footer-dark-link-color: civictheme-color('neutral', 1) !default;
-$civictheme-footer-dark-link-hover-color: civictheme-color('secondary') !default;
-$civictheme-footer-dark-collapsible-border-color: civictheme-color('primary', 3) !default;
+$civictheme-footer-light-background-color: civictheme-color-light('background') !default;
+$civictheme-footer-light-border-color: civictheme-color-light('background') !default;
+$civictheme-footer-light-collapsible-border-color: civictheme-color-light('background') !default;
+$civictheme-footer-light-color: civictheme-color-light('background') !default;
+$civictheme-footer-light-link-color: civictheme-color-light('background') !default;
+$civictheme-footer-light-link-hover-color: civictheme-color-light('interaction-background') !default;
+$civictheme-footer-light-outline-color: civictheme-color-light('background') !default;
+$civictheme-footer-dark-background-color: civictheme-color-dark('interaction-background') !default;
+$civictheme-footer-dark-border-color: civictheme-color-dark('interaction-background') !default;
+$civictheme-footer-dark-collapsible-border-color: civictheme-color-dark('interaction-background') !default;
+$civictheme-footer-dark-color: civictheme-color-dark('background') !default;
+$civictheme-footer-dark-link-color: civictheme-color-dark('background') !default;
+$civictheme-footer-dark-link-hover-color: civictheme-color-dark('interaction-background') !default;
+$civictheme-footer-dark-outline-color: civictheme-color-dark('interaction-focus') !default;
 
 //
 // Header.
 //
+$civictheme-header-desktop-menu-breakpoint: 'm' !default;
 $civictheme-header-logo-desktop-width: 230px !default;
 $civictheme-header-logo-mobile-width: 125px !default;
-$civictheme-header-desktop-menu-breakpoint: 'm' !default;
-$civictheme-header-light-top-color: civictheme-color('neutral', 7) !default;
-$civictheme-header-light-top-background-color: civictheme-color('neutral', 3) !default;
-$civictheme-header-light-middle-background-color: civictheme-color('neutral', 1) !default;
-$civictheme-header-light-middle-border-color: civictheme-color('neutral', 3) !default;
-$civictheme-header-dark-top-color: civictheme-color('neutral', 4) !default;
-$civictheme-header-dark-top-background-color: civictheme-color('primary', 5) !default;
-$civictheme-header-dark-middle-background-color: civictheme-color('primary', 4) !default;
-$civictheme-header-dark-middle-border-color: transparent !default;
 $civictheme-header-sticky-scroll-offset: 100px !default;
```

Comment:
Exception 
$civictheme-header-sticky-scroll-offset
Used in [assets/sass/header](https://github.com/salsadigitalauorg/civictheme_source/blob/feature/CIVIC-973-colour-remap/docroot/themes/contrib/civictheme/assets/sass/header/_header.scss#L14)